src/utils/daos_hdlr.c

```diff
@@ -54,11 +54,278 @@
 
 /* TODO: implement these pool op functions
  * int pool_stat_hdlr(struct cmd_args_s *ap);
- * int pool_get_prop_hdlr(struct cmd_args_s *ap);
- * int pool_get_attr_hdlr(struct cmd_args_s *ap);
- * int pool_list_attrs_hdlr(struct cmd_args_s *ap);
  */
 
+int
+pool_get_prop_hdlr(struct cmd_args_s *ap)
+{
+	daos_prop_t			*prop_query;
+	struct daos_prop_entry		*entry;
+	int				rc = 0;
+	int				rc2;
+
+	assert(ap != NULL);
+	assert(ap->p_op == POOL_GET_PROP);
+
+	rc = daos_pool_connect(ap->p_uuid, ap->sysname,
+			       ap->mdsrv, DAOS_PC_RO, &ap->pool,
+			       NULL /* info */, NULL /* ev */);
+	if (rc != 0) {
+		fprintf(stderr, "failed to connect to pool: %d\n", rc);
+		D_GOTO(out, rc);
+	}
+
+	prop_query = daos_prop_alloc(0);
```

Comment:
Do we need to check if prop_query is NULL?

Reply:
Right ! Done in next commit.

Reply:
Sorry, but on line 81 we should disconnect from the pool, instead of just return. Also, -DER_NOMEM instead of ENOMEM.

Reply:
sure, sorry to not have carefully controlled my last change...